src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form"
+      method="post"
+      data-qa="big"
+      >
+      <img
+          src="images/Search.svg"
+          alt="search"
+          class="icon icon--big"
+        >
```

Comment:
```suggestion
    <form
      action="#"
      class="form"
      method="post"
      data-qa="big"
    >
      <img
        src="images/Search.svg"
        alt="search"
        class="icon icon--big"
      >
```
Please, fix the code style everywhere. Read the checklist more carefully.
![image](https://user-images.githubusercontent.com/102293862/234054934-893567ea-e940-4229-bb21-0ffb147d100f.png)